bootcfg/storage/storagepb/storage.proto

```diff
@@ -25,8 +25,10 @@ message Profile {
   string ignition_id = 3;
   // cloud config id
   string cloud_id = 4;
+  // generic config id
+  string generic_id = 5;
   // support network boot / PXE
-  NetBoot boot = 5;
+  NetBoot boot = 6;
```

Comment:
Changing protocol buffer numeric tags breaks gRPC API backwards compatability. Although I don't know if external users depend on the gRPC API (which will be documented and TLS auth'd with v0.4.0), we use it internally and its best practice to use the next number for new values. See https://developers.google.com/protocol-buffers/docs/proto3 "Updating a Message Type".